.github/workflows/snyk.yml

```
@@ -0,0 +1,103 @@
+        uses: actions/checkout@v4
+
+      - name: Setup Node (for UI)
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Setup Python (for tests tooling)
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+        run: |
```

actionlint (1.7.7)

this step is for running action since it contains at least one of "uses", "with" keys, but also contains "run" key which is used for running shell command

(syntax-check)